plugin/src/main/java/org/opensearch/ml/breaker/MLCircuitBreakerService.java

```diff
@@ -65,18 +78,21 @@ public MLCircuitBreakerService init(Path path) {
         log.info("Registered ML memory breaker.");
         registerBreaker(BreakerName.DISK, new DiskCircuitBreaker(path.toString()));
         log.info("Registered ML disk breaker.");
+        // Register native memory circuit breaker
+        registerBreaker(BreakerName.NATIVE_MEMORY, new NativeMemoryCircuitBreaker(this.osService, this.settings, this.clusterService));
+        log.info("Registered ML native memory breaker.");
 
         return this;
     }
 
     /**
      *
-     * @return the name of any open circuit breaker; otherwise return null
+     * @return any open circuit breaker; otherwise return null
      */
-    public String checkOpenCB() {
+    public ThresholdCircuitBreaker checkOpenCB() {
         for (CircuitBreaker breaker : breakers.values()) {
             if (breaker.isOpen()) {
-                return breaker.getName();
+                return (ThresholdCircuitBreaker) breaker;
```

Comment:
`breaker` class is `CircuitBreaker`, this line enforce it to `ThresholdCircuitBreaker`, seems not so extensible. If we have new child class of `CircuitBreaker`, this line will throw error.

Why need to change the return type from `String` to `ThresholdCircuitBreaker`?

Reply:
Want to get more information like the current threshold when breaker is triggered to help debug.

Reply:
I think we can add a warning log to help